.github/workflows/lint-test.yaml

```diff
@@ -53,3 +53,15 @@ jobs:
 
       - name: Run chart-testing (install)
         run: ct install --target-branch main
+
+      - name: Run KUTTL smoke tests
+        run: |
+          sleep 10s
+          sudo curl -Lo /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.11.0/kubectl-kuttl_0.11.0_linux_x86_64
+          sudo chmod +x /usr/local/bin/kubectl-kuttl
+          export PATH=$PATH:/usr/local/bin
```

Comment:
isn't `/usr/local/bin` already in $PATH?

Reply:
Yes, I will remove this export.